app/controllers/admin/events_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Admin::EventsController < ApplicationController
+  
+  def new
+  end
+
+  def index
+    @events = Event.all
+  end
+
+  def show
+  end
+
+  def create
+    event = Event.new(event_params)
+    if event.save
+      flash[:notice] = 'Your event was successfully created'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.